.github/workflows/branch-to-staging.yml

```
@@ -65,15 +68,63 @@
+      AWS_REGION: ${{ vars.API_REGION_STAGING }}
+    secrets:
+      SERVICE_PAT: ${{ secrets.SERVICE_PAT }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  ihe_stack:
+    if: contains(fromJSON('["ihe_stack", "all"]'), github.event.inputs.jobs_to_run) && ${{ vars.IHE_STACK_NAME }} != null
```

actionlint (1.7.4)

if: condition "contains(fromJSON('[\"ihe_stack\", \"all\"]'), github.event.inputs.jobs_to_run) && ${{ vars.IHE_STACK_NAME }} != null" is always evaluated to true because extra characters are around ${{ }}

(if-cond)